pysimplesoap/client.py

```diff
@@ -754,7 +759,7 @@ def _xml_tree_to_services(self, wsdl, cache, force_download):
 
                 if 'fault_msgs' in op:
                     faults = op['faults'] = {}
-                    for name, msg in op['fault_msgs'].iteritems():
+                    for name, msg in op['fault_msgs'].values():
```

Comment:
Looks strange, you'll get tuple unpacking here and `name, msg` will be incorrect.